packages/itwin/ec3-widget/src/components/ReportConfirmModal.tsx

```diff
@@ -0,0 +1,82 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import {
+  Button,
+  Leading,
+  Modal,
+  ModalButtonBar,
+} from "@itwin/itwinui-react";
+import React, { useState } from "react";
+import "./DeleteModal.scss";
```

Comment:
`ReportConfirmModal` should not import `DeleteModal.scss`.

Reply:
made a seperate file for ```ReportConfirmModal``` css